examples/amp-story/amp-story-desktop-one-panel.html

```diff
@@ -33,6 +33,7 @@
       publisher="AMP Story"
       publisher-logo-src="example.com/logo.png"
       poster-portrait-src="example.com/poster.jpg"
+      desktop-aspect-ratio="9:16" 
```

Comment:
Would something more extreme like `desktop-aspect-ratio="1:1"` be better(or the max width ratio)? This way we can stress test it at it's maximum and more easily see if there is overlap.

It might be worth adding a new example for this and calling it something like amp-story-desktop-aspect-ratio.html

Reply:
Done.